www/style/login.css

```diff
@@ -0,0 +1,15 @@
+.login-form {
```

Comment:
Use this:
```css
#login-form {
  display: block;
  position: relative;
  top: 25%;
  margin: 0 auto;
  width: 20%;
  background-color: #edf4fa;
  border: 1px solid;
  border-color: #e5e6e9 #dfe0e4 #d0d1d5;
  border-radius: 5px;
}

#login-form .mbox.mbox-error {
  background-color: white;
}

#login-form .login-prompt {
  font-weight: bold;
  font-size: 12pt;
  display: inline-block;
  margin-bottom: 18px;
}

#login-form .formElement .form-input {
    width: 100%;
}

```

Reply:
You sure we want it to look like this?

![image](https://user-images.githubusercontent.com/1969470/35514138-835ea9da-0505-11e8-8a87-ba34f3c1fbdf.png)

The prompt for access is getting wrapped. If the window is small, this happens:

![image](https://user-images.githubusercontent.com/1969470/35514168-9d7b4f80-0505-11e8-8940-f6dd62dc7768.png)

Reply:
You can use the `min-width` css property to set a minimal width for the login form.